test/src_test/include/test_discrete.hpp

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2016-2019 Thomas Nagler and Thibault Vatter
+//
+// This file is part of the vinecopulib library and licensed under the terms of
+// the MIT license. For a copy, see the LICENSE file in the root directory of
+// vinecopulib or https://vinecopulib.github.io/vinecopulib/.
+
+#pragma once
+
+#include "gtest/gtest.h"
+#include <vinecopulib/bicop/class.hpp>
+#include <vinecopulib/vinecop/class.hpp>
+
+namespace test_discrete {
+
+using namespace vinecopulib;
+
+TEST(discrete, bicop)
```

Comment:
What annoys me is that we don't really check that the pdf/hfuncs/hinvs are correct, but I reckon that it's kinda hard to do...

Reply:
Yeah, we can remove these parts if you prefer. The test just made it a lot easier to identify bugs during development.

Most stuff is checked implicitly though:
- `Bicop::pdf()` when fitting `Bicop` models,
- `Bicop::hfunc1/2()`  when fitting `Vinecop` models,
- `Bicop::hinv1/2()` always uses numerical inversion when discrete variables are present.